src/handbook/design/process.md

```diff
@@ -16,7 +18,21 @@ been conducted on the high-level UX, and development is not blocked at the start
 of a release sprint. Both visual and UX design iterations will occur throughout the
 first two weeks of a cycle, alongside development efforts.
 
-### Delivery of Art
+### Website and rest of design assets
+
+In contrast to the platform, changes and updates to the web design can be implemented independently and are not limited to specific release cycles. The timing and order of these changes, as well as the creation of other design assets such as blog images, open graphs, presentations, illustrations, etc., depend on prioritization aligned with the company's interests.
+
+## Design Review
+
+This phase aims to ensure that the designs not only look visually appealing, but also effectively communicate the intended message to the audience.
+
+To achieve this, please assess the organization of information, evaluate the readability and coherence of text, and verify the alignment with the project's objectives.
+
+For UX/UI proposals, the design review concludes with the sign-off of Figma designs by the page owner, who is usually the DRI for the Website but may be another team member. As for the rest of the design assets, it's determined by the person who created the Art Request.
```

Comment:
Can we create a screenshot or so in GH? And centralize approvals in one system? Don't want too much information stuck in Figma